### PR TITLE
Add contract call performance tests

### LINKED ISSUE #5320: Extend K6 tests for `api/v1/contracts/call`
### Problem

We need to provide K6 performance tests for `api/v1/contracts/call` REST API invoking different types of pure/view solidity functions.

### Solution

Extends K6 with tests covering pure/view solidity functions.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by mustafauzunn**
**Description**:

K6 performance tests for api/v1/contracts/call REST API invoking different types of pure/view solidity functions.

**Related issue(s)**:

Fixes #5320 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-test/k6/src/web3/test/index.js [hunk 1]
```diff
@@ -21,10 +21,18 @@
...
 
 // import test modules
-import * as ethBlockNumber from './ethBlockNumber.js';
+import * as ethCallBalance from './ethCallBalance.js';
+import * as ethCallSender from './ethCallSender.js';
+import * as ethCallMultiply from './ethCallMultiply.js';
+import * as ethCallIdentifier from './ethCallIdentifier.js';
 
 // add test modules here
-const tests = {ethBlockNumber};
```
Comment (steven-sheehy) [inline hedera-mirror-test/k6/src/web3/test/index.js:27]: Please remove the now unused `ethBlockNumber.js` and `common.js`

File: hedera-mirror-test/k6/src/web3/test/index.js [hunk 2]
```diff
@@ -21,10 +21,18 @@
 import {getSequentialTestScenarios} from '../../lib/common.js';
 
 // import test modules
-import * as ethBlockNumber from './ethBlockNumber.js';
+import * as ethCallBalance from './ethCallBalance.js';
+import * as ethCallSender from './ethCallSender.js';
+import * as ethCallMultiply from './ethCallMultiply.js';
+import * as ethCallIdentifier from './ethCallIdentifier.js';
```
Comment (steven-sheehy) [inline hedera-mirror-test/k6/src/web3/test/index.js:27]: Name these all as `contractCall...js` to match the endpoint name. Same thing for scenario name and other strings in PR. `s/eth/contract/g`

File: hedera-mirror-test/k6/src/web3/test/ethCallSender.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import http from "k6/http";
+
+import {TestScenarioBuilder} from '../../lib/common.js';
+import {isNonErrorResponse} from "./common.js";
+
+const url = __ENV.BASE_URL;
+const contract = __ENV.DEFAULT_CONTRACT_ADDRESS
+
+const payload = JSON.stringify({
+  "to":String(contract),
```
Comment (steven-sheehy) [inline hedera-mirror-test/k6/src/web3/test/ethCallSender.js:30]: Should we have one with `from` populate to a payable?

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-test/k6/src/web3/test/contractCallBalance.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+ * ‍
+ */
+
+import http from "k6/http";
+
+import {TestScenarioBuilder} from '../../lib/common.js';
+import {isNonErrorResponse} from "./common.js";
+
+const url = __ENV.BASE_URL;
+const contract = __ENV.DEFAULT_CONTRACT_ADDRESS
```
Comment (xin-hedera) [inline hedera-mirror-test/k6/src/web3/test/contractCallBalance.js:27]: nit

```suggestion
const contract = __ENV.DEFAULT_CONTRACT_ADDRESS;
```

File: hedera-mirror-test/k6/src/web3/test/contractCallBalance.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+const payload = JSON.stringify({
+  "to":String(contract),
+  "data":"0x6896fabf"
+});
```
Comment (xin-hedera) [inline hedera-mirror-test/k6/src/web3/test/contractCallBalance.js:32]:
```suggestion
const payload = JSON.stringify({
  to: `${contract}`,
  data: '0x6896fabf',
});
```
can run `./gradlew test:spotlessJavascriptApply` to fix most format issues

File: hedera-mirror-test/k6/src/web3/test/contractCallBalance.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+const httpParams = {
+  headers: {
+    'Content-Type': 'application/json',
+  },
+};
```
Comment (xin-hedera) [inline hedera-mirror-test/k6/src/web3/test/contractCallBalance.js:38]: this is required for every web3 test, so we should have a function in `common.js`

```
const jsonPost = (url, payload) => http.post(url, payload, {
  headers: {
    'Content-Type': 'application/json',
  }
});
}
```

**Review by xin-hedera — CHANGES_REQUESTED** [review]

File: hedera-mirror-test/k6/src/web3/test/index.js (hunk 1 above)
Comment (steven-sheehy) [reply hedera-mirror-test/k6/src/web3/test/index.js:27]: I don't see `ethBlockNumber.js` deleted. Did you push this change?

For `common.js`, you're right we should keep except we need to change its implementation to not look at jsonrpc fields.

**Review by IvanKavaldzhiev — APPROVED** [review]
LGTM! I agree with the existing comments.

**Review by xin-hedera — APPROVED** [review]
LGTM

File: hedera-mirror-test/k6/src/web3/test/common.js [hunk 7]
```diff
@@ -15,11 +12,12 @@
...
  * See the License for the specific language governing permissions and
  * limitations under the License.
- * ‍
  */
 
-const errorField = "error";
-const resultField = "result";
+import http from 'k6/http';
+
+const errorField = 'error';
```
Comment (steven-sheehy) [inline hedera-mirror-test/k6/src/web3/test/common.js:19]: Remove json-rpc `error` field and check since it's no longer applicable.

**Pull request merged**